scripts/irods/test/test_native_rule_engine_plugin.py

```diff
@@ -84,6 +84,28 @@ def check_string_count_in_log_section(string, n_occurrences):
                 check_string_count_in_log_section (s,number_of_strings_to_look_for)
 
 
+
+    @unittest.skipIf(plugin_name == 'irods_rule_engine_plugin-python' or test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, 'Native only test when not in a topology')
+    def test_error_smsi_5043 (self):
+        with temporary_core_file() as core:
+            rule_code = rule_texts[self.plugin_name][self.class_name][inspect.currentframe().f_code.co_name]
+            core.add_rule( rule_code )
+            rule_file_text = rule_texts[self.plugin_name][self.class_name][inspect.currentframe().f_code.co_name + "__rule_file"]
+            with tempfile.NamedTemporaryFile(suffix='.r') as rule_file:
+                print (rule_file_text, file = rule_file)
+                rule_file.flush()
+                self.admin.assert_icommand(["irule","-F",rule_file.name], STDOUT_MULTILINE, ["SYS_INTERNAL_ERR = -154000",
+                                                                                             "RULE_ENGINE_CONTINUE = 5000000"])
+                #output, err, rc = self.admin.run_icommand(["irule","-F",rule_file.name])
+                #open('/tmp/dude','w').write( 'rc={rc} err={err} output=\n{output}'.format(**locals()))
+                #self.assertEqual(err.strip(), "true")
+                #self.assertEqual(rc, 0)
+                #self.assertEqual(output.strip(), "SYS_INTERNAL_ERR = -154000 = SYS_INTERNAL_ERR")
```

Comment:
remove if unneeded

Reply:
Agreed, the tests are still a work in progress.